rest-http-secured/nodejs/booster.yaml

```diff
@@ -0,0 +1,8 @@
+source:
+  git:
+    url: https://github.com/bucharest-gold/nodejs-rest-http-secured
+    ref: master
+name: Node.js - REST and RH SSO
+description: A simple HTTP application using Node.js secured by RH SSO
+metadata:
+  runsOn: "none"
```

Comment:
is the `"` necessary?

Reply:
Nope, the latest version doesn't have that anymore :)